textarea/elfinder-sakai/src/webapp/WEB-INF/applicationContext.xml

```diff
@@ -5,54 +5,51 @@
+		<property name="fsService" ref="org.sakaiproject.elfinder.impl.SakaiFsServiceAdapter"/>
+	</bean>
 
-						</bean>
-					</set>
-				</property>
-			</bean>
+	<bean id="org.sakaiproject.elfinder.impl.SakaiFsServiceAdapter"
+		  class="org.sakaiproject.elfinder.impl.FsServiceAdapter">
+		<property name="sakaiFsService" ref="org.sakaiproject.elfinder.SakaiFsService"/>
+		<property name="securityChecker">
+			<bean class="org.sakaiproject.elfinder.impl.SakaiFsSecurityChecker"/>
+		</property>
+		<property name="fsServiceConfig">
+			<bean class="org.sakaiproject.elfinder.impl.SakaiFsServiceConfig"/>
 		</property>
 	</bean>
+
+<!--	<bean id="org.sakaiproject.elfinder.sakai.msgcntr.MsgCntrSiteVolume"-->
+<!--		  class="org.sakaiproject.elfinder.sakai.msgcntr.MsgCntrSiteVolumeFactory"-->
+<!--		  init-method="init">-->
+<!--		<property name="discussionForumManager" ref="org.sakaiproject.api.app.messageforums.ui.DiscussionForumManager"/>-->
+<!--		<property name="messageForumsForumManager" ref="org.sakaiproject.api.app.messageforums.MessageForumsForumManager"/>-->
+<!--		<property name="sakaiFsService" ref="org.sakaiproject.elfinder.sakai.SakaiFsService"/>-->
+<!--		<property name="serverConfigurationService" ref="org.sakaiproject.component.api.ServerConfigurationService"/>-->
+<!--		<property name="uiPermissionsManager" ref="org.sakaiproject.api.app.messageforums.ui.UIPermissionsManager"/>-->
+<!--		<property name="userDirectoryService" ref="org.sakaiproject.user.api.UserDirectoryService"/>-->
+<!--	</bean>-->
+
+<!--	<bean id="org.sakaiproject.elfinder.sakai.samigo.SamSiteVolume"-->
+<!--		  class="org.sakaiproject.elfinder.sakai.samigo.SamSiteVolumeFactory"-->
+<!--		  init-method="init">-->
+<!--		<property name="publishedAssessmentService" >-->
+<!--			<bean class="org.sakaiproject.tool.assessment.services.assessment.PublishedAssessmentService"/>-->
+<!--		</property>-->
+<!--		<property name="sakaiFsService" ref="org.sakaiproject.elfinder.sakai.SakaiFsService"/>-->
+<!--		<property name="serverConfigurationService" ref="org.sakaiproject.component.api.ServerConfigurationService"/>-->
+<!--	</bean>-->
+
+	<!-- Uncomment the following bean definition to enable entity linking to SCORM Player modules. -->
+	<!-- NOTE: this requires the SCORM Player project to be build and deployed to your Tomcat. -->
+	<!--<bean class="org.sakaiproject.elfinder.sakai.scorm.ScormSiteVolumeFactory">
+        <property name="serverConfigurationService" ref="org.sakaiproject.component.api.ServerConfigurationService" />
+        <property name="contentHostingService" ref="org.sakaiproject.content.api.ContentHostingService"/>
+        <property name="securityService" ref="org.sakaiproject.authz.api.SecurityService"/>
+        <property name="siteService" ref="org.sakaiproject.site.api.SiteService" />
+        <property name="scormContentService" ref="org.sakaiproject.scorm.service.api.ScormContentService" />
+    </bean>-->
+
```

Comment:
These comments can be removed now. The samigo provider is now in samigo, and I'll refactor the SCORM one to live in the contrib project, so there won't be a need to add them here any more!